Ukol_1/python/vypocty.py

```diff
@@ -31,7 +34,7 @@ def prumer(cisla: list[int]) -> float:
     Returns:
       Průměrná hodnota čísel.
     """
-    pass
+    return sum(cisla) / len(cisla) if cisla else 0
```

Comment:
v pripade praydenho seynamu prosim vyvolat nasledujici
`raise ValueError('Prazdny seznam')`

Reply:
Teď ho voláte pokaždé i když seznam není prázdný musíte to dát do za podmínky.